src/main/java/io/supertokens/Main.java

```diff
@@ -190,6 +191,11 @@ private void init() throws IOException {
             Cronjobs.addCronjob(this, Telemetry.getInstance(this));
         }
 
+        // starts CleanExpiredAccessTokenSigningKeys cronjob if the access token signing keys can change
+        if (!Config.getConfig(this).getAccessTokenSigningKeyDynamic()) {
```

Comment:
Shouldn't this be `if (Config.getConfig(this).getAccessTokenSigningKeyDynamic()) {`? (remove the `!`)